benchmarks/bvh_driver/CMakeLists.txt

```
@@ -1,9 +1,9 @@
-set(POINT_CLOUDS_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/benchmarks/point_clouds)
+set(BENCHMARKS_UTILS_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/benchmarks/utils)
```

```suggestion
set(ARBORX_BENCHMARKS_UTILS_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/benchmarks/utils)
```
to be on the safe side

---

Actually should we say BENCHMARK[S] without the s?

---

> Actually should we say BENCHMARK[S] without the s?

No strong opinion.